ompi/include/mpi.h.in

```diff
@@ -2507,6 +2508,7 @@ OMPI_DECLSPEC  int PMPI_Imrecv(void *buf, int count, MPI_Datatype type,
                                MPI_Message *message, MPI_Request *request);
 OMPI_DECLSPEC  MPI_Fint PMPI_Info_c2f(MPI_Info info);
 OMPI_DECLSPEC  int PMPI_Info_create(MPI_Info *info);
+OMPI_DECLSPEC  int PMPI_Info_create_env(int argc, char **argv, MPI_Info *info);
```

Comment:
There is a difference between `MPI_Info_create_env` (`char* argv[]`) and `PMPI_Info_create_env` (`char **argv`). Not sure that matters though

Reply:
I agree.  This should be fixed (I note that `MPI_Info_create_env()`, earlier in the file, is correct).